python/pyarrow/_compute.pyx

```diff
@@ -1458,10 +1458,12 @@ class StrptimeOptions(_StrptimeOptions):
     unit : str
         Timestamp unit of the output.
         Accepted values are "s", "ms", "us", "ns".
+    raise_errors : boolean, default True
+        Raise on parsing errors.
```

Comment:
Same here - what's the behavior otherwise?

Reply:
Done.